src/example-types/filters/tagsQuery.js

```diff
@@ -81,9 +81,9 @@ let filter = ({ tags, join, field, exact }) => ({
   },
 })
 
-let buildResultQuery = node => ({
+let buildResultQuery = (node, children = {}, groupsKey = 'tags') => ({
```

Comment:
Cribbed from https://github.com/smartprocure/contexture-elasticsearch/pull/215:
Some more info:

children being spread does the actual composition of child aggregations
groupsKey allows the pivot type to set the name of the aggregation which it uses to handle groups vs columns (which can be next to each other in the same part of the aggregation tree and so need to be uniquely named and also makes it easier to generically traverse the response)